packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -54,6 +55,7 @@ function MapsIndoorsMap(props) {
             useMapProviderModule: false,
             useKeyboard: false, 
             searchAllVenues: false,
+            hideNonMatches: false
```

Comment:
Is this necessary? The atom value is also false as default.

Reply:
Hmmm, it isn't. Removed it. ✅